x-pack/packages/kbn-cloud-security-posture/public/src/hooks/use_get_navigation_url_params.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  CDR_MISCONFIGURATIONS_DATA_VIEW_ID_PREFIX,
+  SECURITY_DEFAULT_DATA_VIEW_ID,
+  CLOUD_SECURITY_POSTURE_PATH,
+} from '@kbn/cloud-security-posture-common';
+import { CoreStart } from '@kbn/core-lifecycle-browser';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { useCallback } from 'react';
+import { CspClientPluginStartDeps } from '../types';
+import { encodeQuery } from '../utils/query_utils';
+import { NavFilter, createFilter } from './use_navigate_findings';
+import { useDataView } from './use_data_view';
+
+/* If we put this on constant folder and import it here, it will cause our bundle size to go above the limit */
+const findingPageNavigationUrl = {
+  findings: {
```

Comment:
let's try not to use term `findings` to the `misconfiguration findings`. `misconfigurations` is a better fit